torchmultimodal/utils/common.py

```diff
@@ -113,3 +113,9 @@ def from_tensor_list(cls, tensor_list, do_round=False):
 
     def __repr__(self):
         return repr(self.tensors)
+
+
+def filter_dict(
```

Comment:
where is this used?

Reply:
Used in the test right now, but in general will be used for state_dict mapping stuff